scripts/bootstrap/compile.sh

```diff
@@ -207,10 +207,24 @@ if [ -z "${BAZEL_SKIP_JAVA_COMPILATION}" ]; then
     cp src/main/java/$i ${OUTPUT_DIR}/classes/$i
   done
 
+  # Create the bazel_tools repository.
+  BAZEL_TOOLS_REPO=${OUTPUT_DIR}/embedded_tools
+  mkdir -p ${BAZEL_TOOLS_REPO}
+  cat <<EOF >${BAZEL_TOOLS_REPO}/WORKSPACE
+worksapce(name = 'bazel_tools')
+EOF
+  ln -s `pwd`/src ${BAZEL_TOOLS_REPO}/src
+  ln -s `pwd`/third_party ${BAZEL_TOOLS_REPO}/third_party
+  ln -s `pwd`/tools ${BAZEL_TOOLS_REPO}/tools
+
+  # Set up @bazel_tools//platforms properly
+  mkdir -p ${BAZEL_TOOLS_REPO}/platforms
+  cp tools/platforms/platforms.BUILD ${BAZEL_TOOLS_REPO}/platforms/BUILD
```

Comment:
why `cp` and not `ln`?

Reply:
...although it won't make a difference on Windows

Reply:
Left it as cp, since the new link_dir function I added only works on directories.